### PR TITLE
String manipulation 2

### PR DESCRIPTION
**Pull request by Jazzmine-Jimenez**
image

### CODE REVIEW

File: string-manipulation-2/src/capitalize-word.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/* eslint-disable no-unused-vars */
+function capitalizeWord(word) {
+  var endOfWord = word.substring(1, word.length);
+  endOfWord = endOfWord.toLowerCase();
+  var firstLetter = word[0].toUpperCase();
+  var newWord = firstLetter + endOfWord;
+
+  if (word.toLowerCase() === 'javascript') {
+    return 'JavaScript';
+  }
```
Comment (thebearingedge) [inline string-manipulation-2/src/capitalize-word.js]: It's best to get this special case out of the way right at the beginning of the function.

File: string-manipulation-2/src/first-chars.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    } else if (tempArray[i] === 'LearningFuze') {
+      newWord = tempArray[i];
+    } else {
+      var endOfWord = tempArray[i].substring(1, tempArray[i].length);
+      endOfWord = endOfWord.toLowerCase();
+      var firstLetter = tempArray[i][0].toUpperCase();
+      newWord = firstLetter + endOfWord;
+    }
+    updatedString = updatedString + ' ' + newWord;
+  }
```
Comment (thebearingedge) [inline string-manipulation-2/src/first-chars.js]: If you slice the string from `0` to `length`, you'll get the first `length` chars.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: string-manipulation-2/src/first-chars.js (hunk 2 above)
Comment (Jazzmine-Jimenez) [reply string-manipulation-2/src/first-chars.js]: Went from 18 lines of code to 4... thats crazy

**Review by thebearingedge — APPROVED** [review]
🎉

**Pull request merged**